docs/source/api/plugin/landing-pages.md

```diff
@@ -234,6 +234,246 @@ If you omit this, the Explorer defaults `includeCookies` to `false` or the curre
 </tbody>
 </table>
 
+## Embedded Explorer landing page
```

Comment:
Should we mention in here that you will need to be on AS3 version 3.X.X to get this option?